packages/devextreme/scss/widgets/base/textArea/_index.scss

```diff
@@ -1,8 +1,10 @@
 // adduse
-
+.dx-texteditor-label{
```

Comment:
We shouldn't add fixed values to the base styles since all themes use these styles. This is only acceptable if these styles are required in all themes. In your case, the problem occurs only with the material theme.